thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/YamlResource.java

```diff
@@ -576,7 +576,7 @@ public Response yamlPreviewApi(
       Map<String, Object> newDetectionConfigMap = new HashMap<>(ConfigUtils.getMap(this.yaml.load(payload)));
       DetectionConfigDTO detectionConfig = buildDetectionConfigFromYaml(tuningStart, tuningEnd, newDetectionConfigMap, null);
       Preconditions.checkNotNull(detectionConfig);
-      detectionConfig.setId(Long.MAX_VALUE);
+      detectionConfig.setId(0L);
```

Comment:
I remember we had some issues before to set this to 0. Could we keep this unchanged?

Reply:
Do you know what issue?
Long.MAX_VALUE is very long in alert message. How about setting it to -1?

Reply:
ok, I reverted first.

Reply:
Setting to 0 will cause fetching all the anomalies from the database in preview. we can keep it for now.